experimental/packages/exporter-trace-otlp-grpc/package.json

```diff
@@ -64,7 +64,7 @@
     "typescript": "4.4.4"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.3"
+    "@opentelemetry/api": "^1.1.0"
```

Comment:
Does this exporter require something that's missing from `api@1.0`?

Reply:
Not directly, but it depends on `"@opentelemetry/sdk-trace-base": "1.1.1"`, which requires peer api version `>=1.1.0`.

Reply:
Right .. I guess that's how it's going to go then! Thanks for all of the work!